index.js

```diff
@@ -92,6 +92,15 @@ class SimpleKeyring extends EventEmitter {
     return Promise.resolve(sig)
   }
 
+  // For eth_decryptMessage:
+  decryptMessage (withAccount, encryptedData) {
```

Comment:
@logvik , I just now (a bit late, as merged) realize that `encryptedData` should be the first parameter here (see sigUtil.decrypt). Should be possibly be revised later (i expect that anyways something else will change).

Reply:
Ignore the above, as the other methods have account/addr parameter first, too, so all fine.